packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -1300,6 +1300,84 @@ module('Integration | operator-mode', function (hooks) {
       .containsText('Jackie Woody Woodster');
   });
 
+  test('can create a new card to add to a linksToMany field from card chooser', async function (assert) {
+    await setCardInOperatorModeState(`${testRealmURL}Person/fadhlan`);
+    await renderComponent(
+      class TestDriver extends GlimmerComponent {
+        <template>
+          <OperatorMode @onClose={{noop}} />
+          <CardPrerender />
+        </template>
+      }
+    );
+
+    await waitFor(`[data-test-stack-card="${testRealmURL}Person/fadhlan"]`);
+    await click('[data-test-edit-button]');
+
+    assert.dom('[data-test-field="friends"] [data-test-pet]').doesNotExist();
+    await click('[data-test-links-to-many="friends"] [data-test-add-new]');
+
+    await waitFor(`[data-test-card-catalog-modal]`);
+    assert
+      .dom('[data-test-card-catalog-create-new-button]')
+      .hasText('Create New Pet');
+    await click('[data-test-card-catalog-create-new-button]');
+
+    await waitFor(`[data-test-stack-card-index="1"]`);
+    await fillIn(
+      '[data-test-stack-card-index="1"] [data-test-field="name"] [data-test-boxel-input]',
+      'Woodster'
+    );
+    await click('[data-test-stack-card-index="1"] [data-test-save-button]');
+    await waitUntil(
+      () => !document.querySelector('[data-test-stack-card-index="1"]')
+    );
+    assert.dom('[data-test-field="friends"]').containsText('Woodster');
+  });
+
+  test('does not create a new card to add to a linksToMany field from card chooser, if user cancel the edit view', async function (assert) {
```

Comment:
I believe the card is created. But the card is just not linked to the parent card. Can you adjust the text to reflect the test.

Reply:
The card is not created if the user doesn't click the save button.